### ISSUE TITLE
Always tries to use the first File as Preview Picture

### ISSUE THREAD

**Issue by Mallo321123** [manyfold3d/manyfold#3010]
**Describe the bug**
When Uploading archives, Manyfold tries to use The first File as a preview, but if that file is (in my case) a Freecad file, it can't generate a preview

**To Reproduce**
1. Upload Archive with .Stl and .Fcstd Files
2. Go to Main Overview
3. Select Latest Models
4. See blank preview

**Comment by Floppy** [comment]
Good call! It will use the first image, or the first file. But, it should use the first *renderable* file as a fallback if there are no images. :+1:
